### ISSUE TITLE
Add Google Pay support

### ISSUE THREAD

**Issue by sneps85** [GreenPassApp/greenpass-app#30]
As for Apple wallet, it would be nice to have the Green Pass also inside Google Pay for faster access. Is that possible?

**Comment by JakobStadlhuber** [comment]
Interesting idea do you know if Google Pay accept .pass formats?

**Comment by djohannes** [comment]
> Interesting idea do you know if Google Pay accept .pass formats?

No, it doesn't. Simply put you have to upload the pass to Google's servers which is a complicated process, and I could only find one other app that offers that: https://forum.xda-developers.com/t/app-5-0-tools-pass2pay-add-your-pkpass-or-pdf-passes-to-google-pay.3952721/ (not open source)

**Comment by usame6666** [comment]
The Google Pay API for Passes provides a secure way to store and access COVID vaccination and test cards on Android-powered devices. To be eligible to use these passes, you must be a healthcare organization, government agency, or an organization authorized by public health authorities to distribute COVID vaccines or tests.

https://developers.google.com/pay/passes/guides/pass-verticals/use-cases?vertical=covid-cards#pass-vertical

**Comment by DRSchlaubi** [comment]
> Interesting idea do you know if Google Pay accept .pass formats?
No, it doesn't and Pass2Pay is an app which can do that as @djohannes mentioned, however the app is cluttered with ads, and it doesn't support covid cards itself

As @usamedev mentioned, Google added an official API especially for this use case; however, it is only accessible by official organizations can request access to it, so I can't make an app myself or PR to anyone else.
That provides additional benefits of displaying more vaccination related information on the card and it being protected by the device's display lock

However, you need to get approved access by following [this](https://support.google.com/googlepay/businessconsole/answer/11044296?visit_id=637776334797066689-853711063&rd=1) guide to get this going

**Comment by DRSchlaubi** [comment]
I/O just introduced a new API for this, I will take a deeper look later

**Comment by DRSchlaubi** [comment]
> Interesting idea do you know if Google Pay accept .pass formats?

It now does, you can find affordable documentation [here](https://codelabs.developers.google.com/add-to-wallet-android#0) and [here](https://developers.google.com/wallet/generic/android?hl=en), you only need to be verified in order to brand your passes
